From 227f95879b6539c2dbf71aa23a56dc02a912bca2 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Wed, 8 Jun 2016 16:40:41 +0200
Subject: [PATCH 3/6] scsi-disk: add missing break

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1465404041-26965-3-git-send-email-pbonzini@redhat.com>
Patchwork-id: 70573
O-Subject: [RHEL7.3 qemu-kvm-rhev PATCH 2/2] scsi-disk: add missing break
Bugzilla: 1338043
RH-Acked-by: Auger Eric <eric.auger@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit ed45cae39117d413155417b57286e9d172c4c411)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/scsi/scsi-disk.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hw/scsi/scsi-disk.c b/hw/scsi/scsi-disk.c
index a00f8d1..4d3d2b7 100644
--- a/hw/scsi/scsi-disk.c
+++ b/hw/scsi/scsi-disk.c
@@ -2744,6 +2744,7 @@ static int32_t scsi_block_dma_command(SCSIRequest *req, uint8_t *buf)
         /* 10-byte CDB.  */
         r->cdb1 = req->cmd.buf[1];
         r->group_number = req->cmd.buf[6];
+        break;
     case 4:
         /* 12-byte CDB.  */
         r->cdb1 = req->cmd.buf[1];
-- 
1.8.3.1

